LINK/usr/bin/miq_postgres_upgrade.sh

```diff
@@ -99,9 +104,41 @@ mount -a
 # init the new cluster
 su - postgres -c "initdb -D ${NEW_PGSQL_DIR}/data-new"
 
+sed -i.bak -e "s/^#*shared_preload_libraries.*/shared_preload_libraries = 'pglogical'/" ${NEW_PGSQL_DIR}/data-new/postgresql.conf
```

Comment:
@Fryguy so do you think it's worth changing these lines (or the ones below) one way or the other?